db/migrate/20230113195033_update_children_to_latest.rb

```diff
@@ -1,8 +1,15 @@
 class UpdateChildrenToLatest < ActiveRecord::Migration[7.0]
   def change
+    return unless table_exists?(:children)
+
     # Do not run this migration on production or on existing data. It is a
     # destructive migration. Remove child records before running.
-    raise "Child records exist, cannot run migration" if Child.any?
+    if ActiveRecord::Base
+      .connection
+      .execute("SELECT EXISTS (SELECT 1 FROM children)")
+      .first["exists"]
+      raise "Child records exist, cannot run migration"
+    end
```

Comment:
Can we just remove this migration? Or do we actually still need it?

Reply:
I've removed it!